recipes/georasters/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "georasters" %}
+{% set version = "0.5.10" %}
+{% set sha256 = "86ad677d76ff71879f7bc6a5aad0d2f60ed826107d5fd2d4705105b8fb4ef648" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - numpy
+    - pandas
+    - docopt
+    - gdal
+    - pyproj
+    - scikit-image
+    - matplotlib
+    - coverage
+    - fiona
+    - geopandas
+    - pysal
+    - affine
+    - rasterstats >=0.12
```

Comment:
Are all of these build requirements?

Reply:
I would not think so, since the package does not need any build. But ``conda skeleton`` generated it like this. 

Any idea why the checks fail in the ``import georasters`` stage, as if it was not built or not being found? Thanks for the help!

Reply:
Please remove the packages which are not build dependencies.

Reply:
Done...the only thing still causing issues is the license file. I had tried 

```python
about:
  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```

but ``linter`` still complained.